src/components/phone/TrainingsCard.vue

```diff
@@ -3,7 +3,7 @@
     <div class="flex flex-row justify-between flex-grow p-3">
       <!--- media/picture --->
       <div class="w-1/6">
-        <img :src="imagePath" alt="Video Preview" class="shadow" />
+        <img :src="imagePath" :title="$t('phoneDashboard.video_preview')" class="shadow" />
```

Comment:
Okay, so for the `ccu-icon` component I added a prop that passes `title` -> `alt`, but for `img` components, it will have to be corrected.
```suggestion
        <img :src="imagePath" :alt="$t('phoneDashboard.video_preview')" class="shadow" />
```